backend/src/parser/TheParser.ts

```diff
@@ -223,6 +224,24 @@ export default class TheParser {
     return `<a href="${encodeURI(decodeURI(url))}" target="_blank">${htmlEscape(decodeURI(url))}</a>`
   }
 
+  processTelegram(url: Url<string>) {
+    if (url.host !== 't.me' && url.host !== 'telegram.me') {
+      return false
+    }
+
+    const match = url.pathname.match(/^\/([^/]+)\/(\d+)/)
+    if (!match) {
```

Comment:
nit: It is always better to be more explicit
```suggestion
    if (match === null) {
```